libs/qlik/src/dashboard-page/stories/vl-qlik-dashboard-page.stories-arg.ts

```diff
@@ -0,0 +1,79 @@
+import { ArgTypes } from '@storybook/web-components';
+import { CATEGORIES, TYPES } from '@domg-wc/common-storybook';
+import { qlikDashboardArgs } from '../../dashboard/stories/vl-qlik-dashboard.stories-arg';
+
+export const qlikDashboardPageArgs = {
+    title: '',
+    url: '',
+    appId: '',
+    exportId: '',
+    selectedView: '',
+    filters: null,
+    views: null,
+};
+
+export const qlikDashboardPageArgTypes: ArgTypes<typeof qlikDashboardPageArgs> = {
+    title: {
+        name: 'title',
+        description: '',
+        table: {
+            type: { summary: TYPES.STRING },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: qlikDashboardPageArgs.title },
+        },
+    },
+    url: {
+        name: 'url',
+        description: '',
+        table: {
+            type: { summary: TYPES.STRING },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: qlikDashboardPageArgs.url },
+        },
+    },
+    appId: {
+        name: 'app-id',
+        description: '',
+        table: {
+            type: { summary: TYPES.STRING },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: qlikDashboardPageArgs.appId },
+        },
+    },
+    exportId: {
+        name: 'export-id',
+        description: '',
+        table: {
+            type: { summary: TYPES.STRING },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: qlikDashboardPageArgs.exportId },
+        },
+    },
+    selectedView: {
+        name: 'selected-view',
+        description: '',
+        table: {
+            type: { summary: TYPES.STRING },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: qlikDashboardPageArgs.selectedView },
+        },
+    },
+    filters: {
+        name: 'filters',
+        description: '',
+        table: {
+            type: { summary: 'Filter[]' },
+            category: CATEGORIES.PROPERTIES,
+            defaultValue: { summary: qlikDashboardArgs.filters },
+        },
+    },
+    views: {
+        name: 'views',
+        description: '',
+        table: {
+            type: { summary: 'Views | DashboardVisualization[][]' },
```

Comment:
Klopt het dat DashboardVisualization een 2D array is of moet dit 
`(Views | DashboardVisualization[])[]` zijn?

Reply:
Dat klopt, Views bevat ook een DashboardVisualization 2D array. Om de opdeling te maken tussen rij en kolom.

Reply:
Done